source/release-notes.txt

```diff
@@ -10,6 +10,27 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.2.0 Changelog
+---------------
+
+*Released September 4, 2023*
+
+New features:
+
+- Ability to change the data type for migrated fields.
+- Ability to control how null values are represented in the migrated data.
+- Ability to use calculated fields for a collection's ``_id`` value.
+- Diagnostic dump includes configuration files to help streamline support. 
+
+Bug fixes:
+
+- Improvements to validation and error handling in mapping rules.
+- Ability to parse a wider range of Oracle DDL files.
+- Fixes an issue where tables were lost when refreshing a schema using SQL Server and Windows authentication. 
+- Fixes an issue where CDC updates may target the wrong document when tables do not have a primary key.
+- Fixes an issue where database passwords can be incorrectly saved to the browser's password library. 
+
+
```

Comment:
[nit] looks like there's an extra newline here!

Reply:
Done, thank you!